code/game/objects/items/devices/radio/beacon.dm

```diff
@@ -75,4 +75,19 @@
 		new /obj/machinery/syndicatebomb( user.loc )
 		playsound(src, 'sound/effects/pop.ogg', 100, 1, 1)
 		qdel(src)
-	return
\ No newline at end of file
+	return
+
+// TERROR SPIDER HOMING BEACON
+
+/obj/item/device/radio/beacon/terrorspider

```

Comment:
make this a subtype of `/obj/item/device/radio/beacon/syndicate` so it doesn't need to define `syndicate = 1`